docker/docker-compose.yml

```diff
@@ -120,10 +131,18 @@ services:
       # todo : we are in process of making them as part of default.
       - PINOT_BROKER_TAG=DefaultTenant
       - PINOT_SERVER_TAG=DefaultTenant
-    restart: on-failure
+    volumes:
+      - ../scripts/wait-for-it.sh:/usr/bin/wait-for-it.sh
```

Comment:
if you put into /usr/bin it should be in the search path right? if not /usr/local/bin? in any case /usr/local/bin is better I think.

Reply:
I tried, but, it doesn't get exec permission. So, I have to use full path.

Reply:
fixed as by committing 755 permission to scripts.